### PR TITLE
Add sampling doc for JS

### PR DESCRIPTION
**Pull request by cartermp**
The first of several follow-ups to https://github.com/open-telemetry/opentelemetry.io/pull/2463

Preview: https://deploy-preview-2482--opentelemetry.netlify.app/docs/instrumentation/js/sampling/

### CODE REVIEW

File: content/en/docs/instrumentation/js/sampling.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Sampling
+weight: 9
+---
+
+[Sampling](/docs/concepts/sampling/) is a process that restricts the amount of
+traces that are generated by a system. the JavaScript SDK offers several
```
Comment (chalin) [inline content/en/docs/instrumentation/js/sampling.md:7]:
```suggestion
traces that are generated by a system. The JavaScript SDK offers several
```

**Comment by svrnm** [comment]
nice! @cartermp on fire:-)

**Review by svrnm — APPROVED** [review]

File: content/en/docs/instrumentation/js/sampling.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+You can also configure the TraceIdRatioBasedSampler in code. Here's an example
+for browser apps:
+
+<!-- prettier-ignore-start -->
+{{< tabpane lang=shell persistLang=false >}}
+
+{{< tab TypeScript >}}
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
+import { TraceIdRatioBasedSampler } from '@opentelemetry/sdk-trace-node';
```
Comment (pichlermarc) [inline content/en/docs/instrumentation/js/sampling.md:75]:
```suggestion
import { WebTracerProvider, TraceIdRatioBasedSampler } from '@opentelemetry/sdk-trace-web';
```

File: content/en/docs/instrumentation/js/sampling.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+const samplePercentage = 0.1;
+
+const provider = new WebTracerProvider({
+    sampler: new TraceIdRatioBasedSampler(samplePercentage),
+});
+{{< /tab >}}
+
+{{< tab JavaScript >}}
+const { WebTracerProvider } = require('@opentelemetry/sdk-trace-web');
+const { TraceIdRatioBasedSampler } = require('@opentelemetry/sdk-trace-node');
```
Comment (pichlermarc) [inline content/en/docs/instrumentation/js/sampling.md:86]:
```suggestion
const { WebTracerProvider, TraceIdRatioBasedSampler } = require('@opentelemetry/sdk-trace-web');
```

**Review by pichlermarc — COMMENTED** [review]
Looks good, thanks for documenting this. :slightly_smiling_face: 
I have two suggestions. :slightly_smiling_face:

File: content/en/docs/instrumentation/js/sampling.md (hunk 2 above)
Comment (cartermp) [reply content/en/docs/instrumentation/js/sampling.md:75]: oof, thanks!

**Pull request merged**